o3po/includes/class-o3po-settings.php

```diff
@@ -1284,6 +1297,7 @@ public function get_all_settings_fields_map() {
                 'doaj_language_code' => 'validate_two_letter_country_code',
                 'custom_search_page' => 'checked_or_unchecked',
                 'page_template_for_publication_posts' => 'checked_or_unchecked',
+                'page_template_abstract_header' => 'trim_settings_field',
```

Comment:
Alternatively, if you don't want the abstract header to be html escaped, please implement and call here a function that ensures that the value is valid html.

Reply:
Abstract heading is a string now.